src/api/dashboard-templates.ts

```diff
@@ -103,6 +113,16 @@ export async function getDashboardTemplates(token: string, type?: LayoutTypes):
   return json.data;
 }
 
+export async function getWidgetMapping(token: string): Promise<WidgetMapping> {
+  const resp = await fetch(`/api/chrome-service/v1/dashboard-templates/widget-mapping`, {
```

Comment:
@Hyperkid123 do you think we need to support the type parameter in this service? IE `landingPage` etc? I'm not sure if we'll have different widget maps for different consumers/pages.

Reply:
This is still an ongoing discussion with UX. I imagine we will require some, but I am not sure to what extend. I'd like to wait a bit on the API side for now.